com.stansassets.build/BuildPipeline/Core/DefaultBuildTasksProvider.cs

```diff
@@ -1,41 +1,78 @@
-using System;
 using StansAssets.Foundation;
-
+using System;
+using UnityEditor.Build;
+
 namespace StansAssets.Build.Pipeline
 {
     class DefaultBuildTasksProvider : IBuildTasksProvider
     {
-        public IBuildTasksContainer GetBuildSteps(IUserEditorBuildSettings buildSettings)
+        public IBuildTasksContainerFull GetBuildSteps(IUserEditorBuildSettings buildSettings)
```

Comment:
`IBuildTasksProvider ` Should only provide steps for the current build system.
`IBuildTasksContainerFull ` - doesn't make much scene in terms of API.
We want to only display those additional default Unity steps in the editor window. No need to change the tool API.